public/stylesheets/store.css

```diff
@@ -138,7 +138,6 @@ fieldset {
     0 4px 6px 0 rgba(112, 157, 199, 0.15);
   border-radius: 4px;
   border: none;
-  overflow: hidden;
```

Comment:
@romain-stripe I had to remove this for the iDEAL Bank dropdown to work. Do you know if this is fine, or might this break something else?

Reply:
Thanks Thor! It was probably added for a reason but can't think of anything specific at the moment, so let's go ahead with this change!